### DISCUSSION TITLE
how to use with fastify?

### DISCUSSION THREAD

**Discussion by capaj** [iway1/trpc-panel#89] [Q&A]
is fastify even supported?

**Comment by laneme** [comment]
They serve the entire trpc-panel in one html string. I believe it can be used with any framework. 

Below code is exactly how I serve it and identical to their [quickstart example](https://github.com/iway1/trpc-panel#quick-start).

```ts
import { renderTrpcPanel } from "trpc-panel";

fastify.get('/panel', (_, res) => {
    return res.header('content-type', 'text/html').send(
        renderTrpcPanel(trpcRoot, {
            url: 'http://localhost:3200',
            transformer: 'superjson',
        })
    )
  })
```
